src/main/resources/com/rultor/agents/req/_head.sh

```diff
@@ -1,5 +1,14 @@
 #!/bin/sh
 
+#@todo #754:30min Remove the deprecation code from the shell.
+# Move the code to Java. May be
```

Comment:
@sebing you wrote "may be" but actually in puzzles we should be sure about the solution